NitroxServer/ConsoleCommands/Processor/ConsoleCommandProcessor.cs

```diff
@@ -54,6 +59,39 @@ public void ProcessCommand(string msg)
             RunCommand(cmd, parts);
         }
 
+        public void ProcessPlayerCommand(string msg, Player player, Perms perms)
+        {
+            if (string.IsNullOrWhiteSpace(msg))
+            {
+                return;
+            }
+
+            string[] parts = msg.Split()
+                .Where(arg => !string.IsNullOrEmpty(arg))
+                .ToArray();
+
+            Command cmd;
+            if (!commands.TryGetValue(parts[0], out cmd))
+            {
+                return;
+            }
+
+            if (cmd.RequiredPermLevel > perms)
+            {
+                player.SendPacket(new ChatMessage(ChatMessage.SERVER_ID, "You do not have permission for this command!"));
+                return;
+            }
+
+            if (cmd.SupportsClientSide)
+            {
+                RunPlayerCommand(cmd, parts, player);
```

Comment:
Do we have the use-cases to support an entirely separate method signature?  I think a command like `/give <item> <player>` would always have player as parameter.  And if it always assumed the executing player you would have to have two different commands in this model. Thoughts?

Reply:
Though i used the different methods due to sending info back to the player without creating a new method in PlayerManager to try getting the player based on the player name